ext/sxml/README

```diff
@@ -5,9 +5,9 @@ It also includes Kirill Lisovsky and Dmitry Lizorkin's sxpath implementation.
   http://ssax.sourceforge.net/
 
  Documents:
-  http://pobox.com/~oleg/ftp/Scheme/SXML-short-paper.html
-  http://pobox.com/~oleg/ftp/Scheme/SXML.html
-  http://pobox.com/~oleg/ftp/Scheme/xml.html
+  http://okmij.org/ftp/Scheme/SXML-short-paper.html
+  http://okmij.org/ftp/Scheme/SXML.html
+  http://okmij.org/ftp/Scheme/xml.html
 
```

Comment:
It used to be that he stated the "official" url is pobox.com/~oleg which redirected to wherever his current page was, but now I don't see that in his site, so maybe he settled in okmij.org permanently.